src/App.jsx

```diff
@@ -14,45 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY_VALUE = 'Sort alphabetically';
+const SORT_BY_LENGTH_VALUE = 'Sort by length';
+const RESET_VALUE = 'Reset';
+const REVERSE_VALUE = 'Reverse';
+const REVERSE_DEFAULT_VALUE = false;
+
+function getGoodsSorted(goods, sortKey, reverse) {
+  const tempGoods = [...goods];
+
+  if (sortKey !== '') {
+    tempGoods.sort((goodA, goodB) => {
+      switch (sortKey) {
+        case SORT_ALPHABETICALLY_VALUE:
+          return goodA.localeCompare(goodB);
+
+        case SORT_BY_LENGTH_VALUE:
+          return goodA.length - goodB.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return (reverse !== REVERSE_DEFAULT_VALUE
+    ? tempGoods.reverse()
+    : tempGoods);
+}
+
+export const App = () => {
+  const [sortSelect, setSortSelect] = useState('');
```

Comment:
I think it's better to name `sortType` or something similar as I have no idea what select we are talking about